web_advanced_search/static/src/js/control_panel/advanced_select_dialog.js

```diff
@@ -22,15 +30,18 @@ odoo.define("web_advanced_search.AdvancedSelectDialog", function (require) {
                 controller_update: function (event) {
                     event.stopPropagation();
                     if (this.$footer) {
+	                    if (event.data.start === true) {
+                           this.$footer.find(".o_use_criteria_button").hide()
+		        	      return
+            	        }
                         const state = event.data.state;
                         const domain = state.getDomain();
-                        let disable_criteria = true;
                         if (domain && domain.length > 0) {
-                            disable_criteria = false;
-                        }
-                        this.$footer
-                            .find(".o_use_criteria_button")
-                            .prop("disabled", disable_criteria);
+                            this.$footer.find(".o_use_criteria_button").prop("disabled", false)
+                            this.$footer.find(".o_use_criteria_button").show()
+                        } else { 
+                            this.$footer.find(".o_use_criteria_button").hide()
```

Comment:
I never see the property disabled set to true. Maybe that is OK. Do we even need to set the property or can we just rely on show() and hide()?

Reply:
show hand hide supercedes "disabled". it is sufficient.
if the element has display = NONE  (wich is what the jquery does) it's enabled/disabled status is irrelevant, it can never be clicked anyway.

Reply:
we need to set the property, because on first load it is disabled == true. if we just showed it it would never switch to false.

Reply:
@NL66278 I also could had marked the button enabled and keep it that way when i do the "Start", few  lines above, but these things can be tricky, there may be a regression where in some circumstance the reload is not full, the start is not triggered (old view) and button remains disabled.
this solution ensures the button to  be enabled.

Reply:
@NL66278 apologies.  In https://github.com/NL66278/web/pull/1/commits/326b8e64897683f35b4dc4d82f05b09138911e75

I solve this in a cleaner way.